src/js/pouchPersisted.js

```diff
@@ -0,0 +1,174 @@
+/*
+Copyright 2016 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.txt
+*/
+
+/* global fluid, floe, PouchDB */
+
+(function ($, fluid) {
+
+    "use strict";
+
+    // Base grade for persistence of model components to Pouch
+    // Implementing grades should ensure that what's integral to
+    // reconstructing the particular instance is stored in the
+    // model
+    fluid.defaults("floe.dashboard.pouchPersisted", {
+        gradeNames: ["floe.dashboard.eventInTimeAware"],
+        model: {
+            // Stores the typename to assist in reconstruction
+            // of stored model components in another context
+            "persistenceInformation": {
+                "typeName": "{that}.typeName"
```

Comment:
What would be a better approach for storing the current model of a component & including some level of metadata that would allow the component to be reconstructed at a later time? Would it make sense to simply persist the entire component as JSON?

It's also possible I'm thinking about this too narrowly, and the capability needed (especially long-term) is an approach to storing model information only, then discovering through some means whether or not a particular component can handle a model with those characteristics for some purpose (display, editing, etc).